core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -35,6 +35,7 @@
 import static com.google.common.base.Preconditions.checkNotNull;
 import static com.google.common.base.Preconditions.checkState;
 
+import com.google.census.CensusContextFactory;
```

Comment:
The is a pure API package called census-core, It currently has dependency on guava and jsr305, but they should be very easy to remove.